lib/html_tag.js

```diff
@@ -1,15 +1,25 @@
 'use strict';
 
-function htmlTag(tag, attrs, text) {
+const encodeURL = require('./encode_url');
+const escapeHTML = require('./escape_html');
+
+function htmlTag(tag, attrs, text, escape = true) {
   if (!tag) throw new TypeError('tag is required!');
 
-  let result = `<${tag}`;
+  let result = `<${escapeHTML(tag)}`;
 
   for (const i in attrs) {
-    if (attrs[i] != null) result += ` ${i}="${attrs[i]}"`;
+    if (attrs[i] === null || typeof attrs[i] === 'undefined') result += '';
+    else {
+      if (i === 'href' || i === 'src') result += ` ${i}="${encodeURL(attrs[i])}"`;
+      else result += ` ${escapeHTML(i)}="${escapeHTML(String(attrs[i]))}"`;
```

Comment:
It's not safe to escape attributes when which value is url.
e.g.
```js
data-url="http://example.com/"
=>
data-url="http:&#x2F;&#x2F;example.com&#x2F;"
```

I think escape `"` is enough.
```js
else result += ` ${escapeHTML(i)}="${String(attrs[i]).replace(/"/g, "&quot;")}"`;
```

```